saleor/payment/gateways/stripe/plugin.py

```diff
@@ -177,13 +188,19 @@ def process_payment(
         payment_method_id = data.get("payment_method_id") if data else None
 
         setup_future_usage = None
+        # DEPRECATED
```

Comment:
Can you add here a note that this should be removed in Saleor 4.0? It will be easier to find this when we will going to remove all deprecated things for 4.0